### DISCUSSION TITLE
Remove Separator Lines

### DISCUSSION THREAD

**Discussion by Alphalaneous** [valinet/ExplorerPatcher#1610] [Ideas]
An option to remove the broken Windows 11 Separator Lines 🥺
![image](https://github.com/valinet/ExplorerPatcher/assets/38200084/3ea6eee8-ce68-4398-989a-7269d02b9fd3)

**Comment by pyrates999** [comment]
I only see separator lines at the top and bottom of Downloads.  Perhaps it's a bug in windows?  I don't see them in explorer when I have libraries enabled.

Are you running a preview release?

Please tell us the windows build number that you're running.
Please also include the explorer patcher version you have installed.

**Reply by Alphalaneous** [reply]
Yeah this is a bug with windows 11 that has no fix. It happens when a library has multiple folders in it. This isn't an issue with explorerpatcher. It happens even without the patcher. That's why I marked this as an idea for a feature to remove these lines. 

To replicate, show libraries in explorer, have one of the libraries point to multiple folders in their properties.

Windows 11 Pro 22H2 Build 22621.1848

**Reply by Alphalaneous** [reply]
Patcher version 22621.1555.55.1

**Reply by pyrates999** [reply]
Explorer patcher is not for fixing bugs in windows.  So I doubt this will be added to it.

**Reply by Alphalaneous** [reply]
It's more of a buggy feature, separators are supposed to exist, but they don't do so right, so the feature I want is just removal of all separators in the navigation pane in general
